api/v1beta1/common_types.go

```diff
@@ -137,6 +137,10 @@ type PodOptions struct {
 	// +optional
 	ServiceAccountName string `json:"serviceAccountName,omitempty"`
 
+	// Should process namespace sharing be enabled on created pods
+	// +optional
+	ShareProcessNamespace *bool `json:"shareProcessNamespace,omitempty"`
```

Comment:
Since the default is false, this probably doesn't need to be a pointer. pointer is only used when we want to differentiate the default value of the variable and the default when users leave the field empty.